darkside-tests/src/utils.rs

```diff
@@ -208,7 +208,7 @@ impl TreeState {
 /// Returns a darkside handler and darkside connector.
 /// Generates a genesis block and adds initial treestate.
 pub async fn init_darksidewalletd(
-    set_port: Option<portpicker::Port>,
+    set_port: Option<zingolib::testutils::portpicker::Port>,
```

Comment:
i dont understand the benefit of re-exporting tempfile and portpicker via zingolib if we are unifying the versions in the workspace anyway. doesnt this just bloat the code and create potential engtangements for no benefit as the versions are already unfied?

Reply:
I was thinking something along the lines of, any project that imports "testutils" should not be using a different version of those dependencies.

It's true that in the workspace the manifest will keep versions the same anyway, but if another project wanted to use `zingolib/testutils` this would give that project a same-version interface for tempfile, and portpicker.